src/api-calls/getAuth.jsx

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+
+const signupHandler = async (email, password) => {
+  console.log(email, password)
+  try {
+    const response = await axios.post(`/api/auth/signup`, {
+      email: email,
+      password: password,
+    });
+    if (response.status === 200 || response.status === 201) {
+      return response.data
+    }
+  } catch (error) {
+    console.log(error);
+    throw error
+  }
+};
+
+
+const loginHandler = async (email, password) => {
+  console.log(email, password)
+  try {
+    const response = await axios.post(`/api/auth/login`, {
+      email: email,
+      password: password,
+    });
+    console.log(response)
+    if (response.status === 200 || response.status === 201) {
+      return response.data
+    }
+  } catch (error) {
+    console.log(error);
```

Comment:
Suggestion : In catch statement, writing `console.error` is a good practice.

Reply:
I will correct it